01_EingabeAusgabeDatentypen.md

```diff
@@ -983,4 +983,37 @@ int main() {
 	return 0;
 }
 ```
-@LIA.eval(`["main.cpp"]`, `g++ -Wall main.cpp -o a.out`, `./a.out`)
\ No newline at end of file
+@LIA.eval(`["main.cpp"]`, `g++ -Wall main.cpp -o a.out`, `./a.out`)
+
+### Quiz
+
+#### Ausgabe
+> Wie lautet der Ausgabeoperator?
+[[<<]]
+
+#### Feldbreite
+> Wie legen Sie die Breite einer Ausgabe auf 10 Zeichen fest? (ohne std:: oder ;)
+[[setw(10)]]
+
+#### Präzision
+> Welcher Ausdruck begrenzt eine Ausgabe auf 2 Stellen nach em Komma? (ohne std:: oder ;)
+[[setprecision(2)]]
+
+#### Escape-Sequenzen
```

Comment:
Könntest Du die Escape Sequenzen noch etwas erläutern? Newline ist für jeden klar, aber bei einem Tab bin ich mir da schon nicht mehr sicher. Vielleicht mit einem Beispiel?

Reply:
Ich habe Beispiele für alle Escape sequenzen in dem entsprechenden Kapitel hinzugefügt. Manche funktionieren nicht in dem eingebundenen Copiler (\b, \r), darum musste ich die Ausgabe fest einfügen.